Code/Framework/AzFramework/AzFramework/Viewport/ViewportBus.h

```diff
@@ -77,4 +77,17 @@ namespace AzFramework
     };
 
     using ViewportBorderRequestBus = AZ::EBus<ViewportBorderRequests>;
+
+    class ViewportImGuiNotifications : public AZ::EBusTraits
+    {
+    public:
+        /*!
+         * Notification that the ImGui menu bar has been activated or deactivated.
+         *
+         * \param active True/False when ImGui is activated/deactivated.
+         */
+        virtual void ImGuiActive(const bool /*active*/){};
```

Comment:
Nit: `//!` is the preferred API comment style.

Reply:
+1, please avoid `/*!` style, just do...

```
//! Notification that the ImGui menu bar has been activated or deactivated.
//! \param active True/False when ImGui is activated/deactivated.
```

suggestion: It might also be clearer to have two separate API calls for `ImGuiActivated` and `ImGuiDeactivated` or maybe have the name be `ImGuiStateChanged`?

Also please put the braces on a new line and remove the trailing `;`, it's not needed here

```
virtual void ImGuiActived([[maybe_unused]] bool active)
{
}
```